bin/load_db_scxa_dimred.sh

```diff
@@ -13,6 +13,7 @@ EXP_ID=${EXP_ID:-$2}
 DIMRED_TYPE=${DIMRED_TYPE:-$3}
 DIMRED_FILE_PATH=${DIMRED_FILE_PATH:-$4}
 DIMRED_PARAM_JSON=${DIMRED_PARAM_JSON:-$5}
+#SCRATCH_DIR=${SCRATCH_DIR:-"$(dirname ${DIMRED_FILE_PATH})"}
```

Comment:
Could you please remove this commented line?
Thanks

Reply:
Done in a71412ddbbad1d7721d658a277ee9257e3efb576